src/components/DatePicker/index.tsx

```diff
@@ -83,6 +85,7 @@ export const DatePicker = ({
   resetLabel,
   onChange,
   onReset,
+  withFrontAdornment = true,
```

Comment:
Could you rename it to `withLeftAdornment`? To be consistent with all other adornments?